library/testing/TestSDKLoader.cxx

```diff
@@ -206,5 +206,16 @@ int TestSDKLoader(int argc, char* argv[])
     return EXIT_FAILURE;
   }
 
+  // Load from memory
+  try
+  {
+    load.loadGeometry({ 0.f, 0.f, 0.f, 0.f, 1.f, 0.f, 1.f, 0.f, 0.f }, { 0, 1, 2 });
```

Comment:
we may need a TestDSKLoadGeometryFromMemory that do actual image comparison. I know there is a python test but python tests should be considered optional and focus on python bindings.